snapd.go

```diff
@@ -523,6 +524,14 @@ func action(ctx *cli.Context) {
 							}).Error(err)
 						}
 					}
+					if err != nil {
```

Comment:
For this to be an error we would want this to be:
```go
// Check if signature file not found and plugin trust is enabled
if err != nil && cfg.Control.PluginTrust > 0 {
```
Should probably also add some context to the error about it being a failure to read the signature file.